### PR TITLE
chore: add separate eslint package

### PR DESCRIPTION
**Pull request by ErnoW**
---
name: 'Pull request'
about: A new pull request
---

### Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/MoralisWeb3/Moralis-JS-SDK/blob/main/SECURITY.md).
- [x] My code is conform the [code style](https://github.com/MoralisWeb3/Moralis-JS-SDK/blob/main/CODE_STYLE.md)
- [x] I have made corresponding changes to the documentation
- [x] I have updated Typescript definitions when needed

### Issue Description

- Move eslint rules to separate package
- Add common rules (based on google and airbnb preset without too much opinionated rules)
- Include this ruleset in all packages and demo projects

### CODE REVIEW

File: packages/eslint/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# @moralisweb3/eslint-config
```
Comment (b4rtaz) [inline packages/eslint/README.md:1]: I think the name of this directory is not consistent. The package is `eslint-config`, the directory is `eslint`. Should be `eslintConfig`.

`packages/eslintConfig/README.md`

**Comment by b4rtaz** [comment]
Just one remark. The rest looks fine.

**Review by b4rtaz — CHANGES_REQUESTED** [review]

**Review by sogunshola — DISMISSED** [review]
Looks good

**Comment by b4rtaz** [comment]
`This branch has conflicts that must be resolved`

**Review by b4rtaz — APPROVED** [review]

**Pull request merged**